code/game/dna/genes/monkey.dm

```diff
@@ -7,7 +7,7 @@
 /datum/dna/gene/monkey/can_activate(var/mob/M,var/flags)
 	return ishuman(M) || istype(M,/mob/living/carbon/monkey)
 
-/datum/dna/gene/monkey/activate(var/mob/living/M, var/connected, var/flags)
+/datum/dna/gene/monkey/activate(mob/living/carbon/human/M, connected, flags)
```

Comment:
in future dont change stuff like this unless you're doing the entire file or changing the proc.

Reply:
Alright.